tests/integration_tests/event/test_delete.py

```diff
@@ -17,6 +30,21 @@ def test_delete_event(client: TestClient, data_db: Session, user_header_token):
     )
 
 
+def test_delete_event_unfccc(
+    client: TestClient, data_db: Session, non_cclw_user_header_token
+):
+    setup_db(data_db)
+    response = client.delete(
+        "/api/v1/events/E.0.0.3", headers=non_cclw_user_header_token
+    )
+    assert response.status_code == status.HTTP_200_OK
+    assert data_db.query(FamilyEvent).count() == 2
+    assert (
+        data_db.query(FamilyEvent).filter(FamilyEvent.import_id == "E.0.0.3").count()
```

Comment:
You could also call an endpoint or function declared in the backend as well to validate that the system behaves correctly?